.circleci/config.yml

```diff
@@ -36,9 +36,36 @@ jobs:
       - run:
           name: Push image
           command: gcloud docker -- push us.gcr.io/model-159019/gh:$(cat /tmp/tag)
-
+  functional-test:
+    docker:
+      - image: google/cloud-sdk
+    working_directory: ~/repo
+    steps:
+      - checkout
+      - run: git submodule sync
+      - run: git submodule update --init
+      - run: &setup_gcloud
+          name: Setup gcloud
+          command: |
+            echo ${GCLOUD_SERVICE_KEY} | base64 -d > ${HOME}/gcp-key.json
+            gcloud auth activate-service-account --key-file ${HOME}/gcp-key.json
+            gcloud --quiet config set project model-159019
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - run:
+          name: Install apt packages
+          command: apt-get install -y wget jq
```

Comment:
Just out of curiosity, what is `jq` used for here?

Reply:
I _was_ going to parse the responses with jq and make some assertions about them, but it turned out not to be super straightforward, and I figured that making detailed assertions about response objects was best left for other tests (i.e. literally anything besides a bash script).

Reply:
Ahh, that makes perfect sense! Good to know we could check `grpcurl` results this way. Maybe in the future we can unify the unit tests we have that check results from individual queries with checks we make here (eg, have expected responses stored in the repo in JSON format that both the unit tests + this bash-based test can check against with less friction)